sql/core/src/test/scala/org/apache/spark/sql/JoinHintSuite.scala

```diff
@@ -489,10 +489,16 @@ class JoinHintSuite extends PlanTest with SharedSparkSession with AdaptiveSparkP
         assertShuffleHashJoin(
           sql(equiJoinQueryWithHint("SHUFFLE_HASH(t1, t2)" :: Nil)), BuildLeft)
         assertShuffleHashJoin(
-          sql(equiJoinQueryWithHint("SHUFFLE_HASH(t1, t2)" :: Nil, "left")), BuildRight)
+          sql(equiJoinQueryWithHint("SHUFFLE_HASH(t1, t2)" :: Nil, "left")), BuildLeft)
```

Comment:
Why change this?

Reply:
In this situation, t1 is smaller than t2, so it now picks t1.  Before it was not possible to pick t1 and so t2 was picked.

Reply:
Yes, I meant that the original test coverage (`BuildRight`) is removed.